### DISCUSSION TITLE
With selfhosted enabled, will cameras still work without a wifi connection?

### DISCUSSION THREAD

**Discussion by jmbulan** [gtxaspec/wz_mini_hacks#595] [Q&A]
wzminihacs loaded onto two of my spare v3s mounted as a dashcam.
 i have self hosted mode enable and mp4 write. 

my issues is when i recently went to review footage alot of recordings were missing. not sure if its cause when i leave home they still reach my network and then fail when i start the car again leaving  the parking lot?

**Comment by endertable** [comment]
How are you dealing with the date issue?  When cams reboot without network they use a date setup by last good boot.  So you could be overwriting files?!?

**Reply by jmbulan** [reply]
I haven't until reading this comment lol. That must be where my missing files are at...  thanks I'll have to follow up

And if that's the case then... Files wouldn't get over written as past files are just waaaay more in the past right?
